src/sentry/sentry_metrics/indexer/redis_mock.py

```diff
@@ -0,0 +1,150 @@
+import hashlib
+from typing import Any, Dict, List, Optional, Union
+
+from django.conf import settings
+
+from sentry.utils.redis import redis_clusters
+
+from .base import StringIndexer
+
+
+def get_client() -> Any:
+    return redis_clusters.get(settings.SENTRY_METRICS_INDEXER_REDIS_CLUSTER)
+
+
+def get_int(string: str) -> int:
+    return int.from_bytes(hashlib.md5(string.encode("utf-8")).digest(), "big") % (10 ** 8)
```

Comment:
Another nit, but you probably just want to take 8 bytes, right?

```suggestion
    return int.from_bytes(hashlib.md5(string.encode("utf-8")).digest()[:8], "big")
```